packages/devtools_app/lib/src/screens/performance/panes/timeline_events/timeline_events_controller.dart

```diff
@@ -235,9 +231,8 @@ class TimelineEventsController extends PerformanceFeatureController
     // available in the engine.
     int? uiThreadId;
     int? rasterThreadId;
-    for (TraceEvent event in threadNameEvents) {
-      final name = event.args!['name'] as String;
-
+    for (ThreadNameEvent event in threadNameEvents) {
+      final name = event.name!;
```

Comment:
Is name guaranteed to be set? If so maybe it shouldn't be optional

Reply:
As far as I know, the name should always be present.